lib/smart_proxy_pulp_plugin/pulp_api.rb

```diff
@@ -35,5 +35,14 @@ class Api < Sinatra::Base
       content_type :json
       {:puppet_content_dir => ::PulpProxy::Settings.settings.puppet_content_dir}.to_json
     end
+
+    get '/status/settings' do
+      content_type :json
+      settings = {
```

Comment:
Should we add the `puppet_content_dir` too and deprecate the `/status/puppet` endpoint?

Reply:
I suppose that makes sense.